src/components/application_manager/include/application_manager/hmi_capabilities_impl.h

```diff
@@ -78,6 +78,11 @@ class HMICapabilitiesImpl : public HMICapabilities {
 
   void set_ccpu_version(const std::string& ccpu_version) OVERRIDE;
 
+  bool matches_ccpu_version(const std::string& ccpu_version) OVERRIDE;
+
+  void software_version_received(
```

Comment:
@ydementieiev Function should return `bool`?

Reply:
@AByzhynar [renamed](https://github.com/smartdevicelink/sdl_core/pull/3269/commits/0d60c4e63814fd53aa0e6c9e0f26893e9587efca#diff-5655615f464bf3d9794062f13d8749caR83)